frontend/src/routes/_protected/treecluster/index.tsx

```diff
@@ -69,15 +77,37 @@ function Treecluster() {
 
       <section className="mt-10">
         <div className="flex justify-end mb-6 lg:mb-10">
-          <Dialog
-            headline="Bewässerungsgruppen filtern"
-            fullUrlPath={Route.fullPath}
-            onApplyFilters={() => handleFilter()}
-            onResetFilters={() => setFilteredData(clustersRes.data)}
-          >
-            <StatusFieldset />
-            <RegionFieldset />
-          </Dialog>
+          <div className="flex items-center gap-4">
```

Comment:
Please do not use so many `<div>` elements. Let's make sure that we don't create unnecessary HTML elements and also use semantic HTML. That would be great 😍 

Maybe we can add the input field into a new component? It should be reusable.